bitnami/external-dns/templates/deployment.yaml

```diff
@@ -290,6 +291,27 @@ spec:
             {{- end }}
           {{- end }}
           env:
+          # Akamai environment variables
+            {{- if .Values.akamai.host }}
+            - name: EXTERNAL_DNS_AKAMAI_SERVICECONSUMERDOMAIN
+              value: {{ .Values.akamai.host }}
+            {{- end }}
+            {{- if .Values.akamai.accessToken }}
+            - name: EXTERNAL_DNS_AKAMAI_ACCESS_TOKEN
+              value: {{ .Values.akamai.accessToken }}
+            {{- end }}
+            {{- if .Values.akamai.clientToken }}
+            - name: EXTERNAL_DNS_AKAMAI_CLIENT_TOKEN
+              value: {{ .Values.akamai.clientToken }}
+            {{- end }}
+            {{- if or (.Values.akamai.clientSecret) (.Values.akamai.secretName) }}
+            - name: EXTERNAL_DNS_AKAMAI_CLIENT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "external-dns.secretName" . }}
+                  key: akamai_client_secret
+            {{- end }}
+            {{- end }}
```

Comment:
I think it is missing here an opening `{{- if eq .Values.provider "akamai" }}`, the number of if/end does not match.

Reply:
Looks like I had a merge error and the beginning "if" got separated from the end.  Fixed now.